poolrpc/trader.proto

```diff
@@ -201,6 +201,33 @@ service Trader {
             get: "/v1/pool/node_ratings"
         };
     };
+
+    /*
+    BatchSnapshots returns all snapshots in the range of the start batch ID and
+    end batch ID. Start and end are interpreted in the same way as the output
+    of the BatchHistory is returned: The first batch (start batch ID) must be
+    the OLDEST batch to return. The last batch (end batch ID) must be the NEWEST
+    batch to return. A maximum of 100 snapshots can be queried in one call, if
+    the start and end batch ID are further apart than that, an error is
+    returned.
+    */
+    rpc BatchSnapshots (BatchSnapshotsRequest)
+        returns (BatchSnapshotsResponse) {
+        option (google.api.http) = {
+            get: "/v1/pool/batch/snapshots/{start_batch_id}/{end_batch_id}"
+        };
+    };
+
+    /* pool: `auction history`
+    BatchHistory returns a list of all batches that were ever created. The list
+    of batch IDs is sorted in ascending order, meaning the oldest batch is the
+    first in the list, while the most recent batch is the last entry.
+    */
+    rpc BatchHistory (BatchHistoryRequest) returns (BatchHistoryResponse) {
```

Comment:
Why are both these RPCs needed?  I was thinking something like `{end_batch, num}` would be enough to paginate backwards through the batch history avoid multiple round trips.

Reply:
Hmm, yeah, makes sense. Not sure why I came up with the other pattern...